### ISSUE TITLE
`ava/use-test` does not allow `anyTest` in `.cts` and `.mts` files

### ISSUE THREAD

**Issue by gebsh** [avajs/eslint-plugin-ava#353]
Currently, the `ava/use-test` allows naming the default import `anyTest` instead of `test` in TypeScript files:

> In a TypeScript file (`.ts` or `.tsx`) AVA can be assigned to a variable named `anyTest` in order to define the types of `t.context `(see [Typing t.context](https://github.com/avajs/ava/blob/main/docs/recipes/typescript.md#typing-tcontext)).

However, TS 4.7 introduced [new file extensions](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-4-7.html#new-file-extensions), `.cts` and `.mts`, that are not recognized by this rule.

For example, the following import statement does not report any violations when placed in a file `foo.ts`:

```ts
import anyTest from 'ava';
```

Renaming the file to `foo.cts` or `foo.mts` produces the following warning, though:

```txt
AVA should be imported as `test`.
```
